src/urllib3/request.py

```diff
@@ -78,11 +85,16 @@ def request(
             )
         else:
             return self.request_encode_body(
-                method, url, fields=fields, headers=headers, **urlopen_kw
+                method, url, fields=fields, headers=headers, **urlopen_kw  # type: ignore
```

Comment:
```
error: Argument 5 to "request_encode_body" of "RequestMethods" has incompatible type "**Dict[str, str]"; expected "bool"
```
I think it's related to [this issue](https://github.com/python/mypy/issues/5382)